eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -279,9 +268,14 @@ extends:
                     -PRBody "$(GeneratedSDKInformation)"
                     -OpenAsDraft $true
 
-            - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
-              - template: /eng/common/pipelines/templates/steps/detect-api-changes.yml
-                parameters:
-                  ArtifactPath: $(GeneratedSDK.StagedArtifactsFolder)
-                  ArtifactName: $(PipelineArtifactsName)
-                  RepoRoot: $(SdkRepoDirectory)
+          - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+            - pwsh: |
+                . $(SpecRepoDirectory)/eng/common/scripts/Helpers/ApiView-Helpers.ps1
+                Create-API-Review `
+                  -specGenSDKArtifactPath "$(System.DefaultWorkingDirectory)/$(SpecGenSdkArtifactPath)/$(SpecGenSdkArtifactName).json" `
```

Comment:
Both `SpecGenSdkArtifactPath` variable and `SpecGenSdkArtifactName` variable might not be set in the previous steps. It needs the existence check in the `helpers.ps1` file.